### PR TITLE
Adjust env name

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: metrics-exporter/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -168,7 +168,7 @@ spec:
                 configMapKeyRef:
                   name: prometheus
                   key: instance-metrics-exporter-infra
-            - name: METRICS_EXPORTER_INFRA_PROMETHEUS_INSTANCE
+            - name: MANAGEMENT_API_PROMETHEUS_INSTANCE
```
Comment (harshad16) [inline metrics-exporter/base/deploymentconfig.yaml:171]: seems like 'a' is missing in management-api in here: https://github.com/thoth-station/thoth-application/blob/80f1ccbce69ac15bbac034ed5c18e56b553cdbf0/metrics-exporter/base/deploymentconfig.yaml#L175
i missed this in previous pr review

Comment (harshad16) [reply metrics-exporter/base/deploymentconfig.yaml:171]: i guess its correct : https://github.com/thoth-station/thoth-application/blob/80f1ccbce69ac15bbac034ed5c18e56b553cdbf0/core/overlays/stage/configmaps.yaml#L52

Comment (pacospace) [reply metrics-exporter/base/deploymentconfig.yaml:171]: I updated it to be correct in English :D

**Review by harshad16 — APPROVED** [review]
thanks for the changes. :+1:

**Pull request merged**